src/styling/Home.css

```diff
@@ -0,0 +1,39 @@
+.container {
+  padding: 0;
+  display: grid;
+  grid-template-columns: 1fr;
+  margin: 0 auto;
+  background-color: #ec4c8b;
+}
```

Comment:
- [ ] The layout of the country cards should be the same as the one in the given design to ensure requirements are met. It should be in two columns instead of one, kindly fix it.